devito/ops/compiler.py

```diff
@@ -30,3 +53,122 @@ def prepare_ops(self, soname, ccode, hcode):
         else:
             warning("Couldn't find OPS_INSTALL_PATH \
                 environment variable, please check your OPS installation")
+
+    def cuda_compiler(self, soname):
```

Comment:
nitpicking: about about calling `jit_cuda` ? `cuda_compiler` makes me think of something else. Otherwise, `compile_cuda`...

Reply:
also, this should probably be a private method? as the actual API is still `jit_compile`

Reply:
I agree. `compile_cuda` is better than `cuda_compiler`.
About the private - **I'm not sure**, but doesn't that go against Python's philosophy? (In this case)

Reply:
it's never going to be used by the outside world right? so it should be private

I don't think this has anything to do with python and its philosophy. Maybe you wanna elaborate what you meant?

Reply:
Python does not have the concept of private methods because the language has the philosophy: – “_we are all adults consenting here_”. (The phrase commonly used).

Since I do not have the habit of using private things, my goal here is just to align this decision to future similar cases. 

I will change to private.